app/controllers/event/qualifications_controller.rb

```diff
@@ -37,7 +37,8 @@ def update
   def entries
     types = event.role_types
     @leaders ||= participations(*types.select(&:leader?))
-    @participants ||= participations(*types.select(&:participant?))
+    @participants ||= participations(*types.select(&:participant?)) +
+                      participations(Event::Role::Helper) # According to https://github.com/hitobito/hitobito_jubla/issues/33
```

Comment:
Wäre es nicht besser, das im Wagon customizable zu machen? Oder wenigstens abhängig von `kind == :participant` (so wie wir bereits `participant?` und `leader?` haben könnten wir auch `helper?` implementieren)? Dann könnten Wagons, welche anders mit Helfenden umgehen, das overriden.

Reply:
`helper` kind gibts schon
Da sind aber halt auch andere Rollen wie `Event::Role::Cook` dabei..
Und es scheint auf mich eh ziemlich custom welche von diesen Rollen Qualifikationen erhalten: https://github.com/hitobito/hitobito_pbs/issues/233

Ist die Frage ob man das als `self.additional_qualification_event_roles` klassenattribut oder so steuern möchte :thinking:

Reply:
Wird `helper` schon woanders verwendet? Mir scheint eher, als wäre der `Event::Role::Cook` nicht `helper` in diesem Sinne, wenngleich natürlich hilfreich. Könnte es sinnvoll sein, hier ein `support` oder so einbauen, und die Küchenhelfer dem zuzuordnen. Es gibt sicherlich öfter Rollen, die eher eine Hintergrundrolle für den Kurs innehaben, aber nicht direkt daran teilnehmen. Allenfalls müsste man dem Koch, der auch die Verlängerung bekommen soll, eben dann auch die Rolle "Helfer" geben... Oder verstehe ich das falsch?